lang/python/Jinja2/Makefile

```diff
@@ -5,19 +5,18 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=Jinja2
-PKG_VERSION:=2.10
+PKG_VERSION:=2.10.1
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/J/Jinja2
-PKG_HASH:=f84be1bb0040caca4cea721fcbbbbd61f9be9464ca236387158b0feea01914a4
+PKG_HASH:=065c4f02ebe7f7cf559e49ee5a95fb800a9e4528727aec6f24402a5374c65013
+
 PKG_BUILD_DEPENDS:=python3
```

Comment:
Can you try removing this line? Since python3-light is already a dependency, this should be unnecessary.

Reply:
I will run the full build for all our (Turris) platforms to see what it does. If I build it locally for just one package,  it won't show anything.

Reply:
It seems good!